vsphere/panels.libsonnet

```diff
@@ -0,0 +1,1118 @@
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Disk throughput')
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('KiBs')
+          ),
+        ])
+        +
+        table.queryOptions.withTransformationsMixin([
+          {
+            id: 'joinByField',
+            options: {
+              byField: 'vcenter_host_name',
+              mode: 'outer',
+            },
+          },
+          {
+            id: 'filterFieldsByName',
+            options: {
+              include: {
+                names: [
+                  'vcenter_host_name',
+                  'Value #A',
+                  'Value #B',
+                  'Value #C',
+                  'Value #D',
+                  'Value #E',
+                  'vcenter_vm_name 1',
+                ],
+              },
+            },
+          },
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {},
+              includeByName: {},
+              indexByName: {
+                'Value #A': 2,
+                'Value #B': 3,
+                'Value #C': 4,
+                'Value #D': 5,
+                'Value #E': 6,
+                vcenter_host_name: 0,
+                'vcenter_vm_name 1': 1,
+              },
+              renameByName: {
+                'Value #A': 'CPU utilization',
+                'Value #B': 'Memory utilization',
+                'Value #C': 'Memory ballooned',
+                'Value #D': 'Memory swapped',
+                'Value #E': 'Disk throughput',
+                'Value #F': 'Memory swapped',
+                vcenter_cluster_name: 'Cluster',
+                'vcenter_cluster_name 1': 'Cluster',
+                vcenter_host_name: 'Host',
+                'vcenter_host_name 1': 'Host',
+                vcenter_vm_name: 'Name',
+                'vcenter_vm_name 1': 'Name',
+                'vcenter_vm_name 2': '',
+              },
+            },
+          },
+        ]),
+
+      clusterCPUEffective:
+        barGauge.new(title='Cluster CPU effective')
+        + barGauge.queryOptions.withTargets([
+          t.clusterCPUEffective,
+        ])
+        + barGauge.panelOptions.withDescription('The effective CPU capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.withUnit('rotmhz')
+        + barGauge.standardOptions.color.withMode('continuous-YlRd'),
+
+      clusterCPULimit:
+        barGauge.new(title='Cluster CPU limit')
+        + barGauge.queryOptions.withTargets([
+          t.clusterCPULimit,
+        ])
+        + barGauge.panelOptions.withDescription('The available CPU capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.withUnit('rotmhz')
+        + barGauge.standardOptions.color.withMode('continuous-YlRd'),
+
+      clusterCPUUtilization:
```

Comment:
Utilization panels should be 0-100% with blue-yellow-red-gradient. Applies to:
- `Cluster CPU utilization`
- `Cluster memory utilization`
- `Top CPU utilization by cluster`
- `Top memory utilization by cluster`
- `Top CPU utilization ESXi hosts`
- `Top memory utilization ESXi hosts` 
- `Host CPU utilization`
- `Host memory utilization`
- `CPU utilization` 
- `Memory utilization`

Reply:
I've gone ahead and changed each one of these to use the `commonlib.panels.memory.timeSeries.usagePercent`, so they should all be standardized now.